sass/components/_file.scss

```diff
@@ -225,29 +225,76 @@ Style guide: file.3
 */
 .file--link {
   background-color: transparent;
+  @extend .col-md-10;
+  @extend .col-sm-10;
+  @extend .col-xs-10;
+  border-top: 1px solid $ec-grey15;
+  &:last-child {
+    border-bottom: 1px solid $ec-grey15;
+  }
+  padding: 1em 0;
+  margin-bottom: 0;
+  .file__title {
 
-  .file__metadata--link {
```

Comment:
is this really needed? I mean, better to avoid enforcing bootstrap dependencies in new components we define.